src/renderer/components/ft-icon-button/ft-icon-button.js

```diff
@@ -55,60 +54,26 @@ export default Vue.extend({
   },
   data: function () {
     return {
-      dropdownShown: false,
-      id: ''
+      dropdownShown: false
     }
   },
-  mounted: function () {
-    this.id = `iconButton${this._uid}`
-  },
   methods: {
-    toggleDropdown: function () {
-      const dropdownBox = $(`#${this.id}`)
-
-      if (this.dropdownShown) {
-        dropdownBox.get(0).style.display = 'none'
-        this.dropdownShown = false
-      } else {
-        dropdownBox.get(0).style.display = 'inline'
-        dropdownBox.get(0).focus()
-        this.dropdownShown = true
-
-        dropdownBox.focusout(() => {
-          const shareLinks = dropdownBox.find('.shareLinks')
-
-          if (shareLinks.length > 0) {
-            if (!shareLinks[0].parentNode.matches(':hover')) {
-              dropdownBox.get(0).style.display = 'none'
-              // When pressing the profile button
-              // It will make the menu reappear if we set `dropdownShown` immediately
-              setTimeout(() => {
-                this.dropdownShown = false
-              }, 100)
-            }
-          } else {
-            dropdownBox.get(0).style.display = 'none'
-            // When pressing the profile button
-            // It will make the menu reappear if we set `dropdownShown` immediately
-            setTimeout(() => {
-              this.dropdownShown = false
-            }, 100)
-          }
-        })
-      }
-    },
-
-    focusOut: function () {
-      const dropdownBox = $(`#${this.id}`)
-
-      dropdownBox.focusout()
-      dropdownBox.get(0).style.display = 'none'
+    // used by the share menu
+    hideDropdown: function () {
       this.dropdownShown = false
     },
 
     handleIconClick: function () {
       if (this.forceDropdown || (this.dropdownOptions.length > 0)) {
-        this.toggleDropdown()
+        this.dropdownShown = !this.dropdownShown
+
+        if (this.dropdownShown) {
+          // wait until the dropdown is visible
+          // then focus it so we can hide it automatically when it looses focus
```

Comment:
Haven't tested yet but noticed a typo in the comment
```suggestion
          // then focus it so we can hide it automatically when it loses focus
```